release 2.32.0 (#1206)
microsoft/onefuzz: CHANGELOG.md

> * Agent: Changed `azcopy` calls to always retry when copying files generates a "source modified during transfer" error.  [#1196](https://github.com/microsoft/onefuzz/pull/1196)

Reviewer: ```suggestion
* Agent: Changed `azcopy` calls to always retry when source files are modified mid-copy.  [#1196](https://github.com/microsoft/onefuzz/pull/1196)
```

* Agent: Continued development related to upcoming features.  [#1146](https://github.com/microsoft/onefuzz/pull/1146)

> * Agent: Logs from `azcopy` failures now redact SAS urls.  [#1194](https://github.com/microsoft/onefuzz/pull/1194)

Reviewer: ```suggestion
* Agent: SAS URLs are now redacted in logged `azcopy` failures.  [#1194](https://github.com/microsoft/onefuzz/pull/1194)
```

> * CLI: Include the number of VMs used per-task to `onefuzz status top`.  [#1169](https://github.com/microsoft/onefuzz/pull/1169)

Reviewer: Fixing my own suggestion:
```suggestion
* CLI: Include the number of VMs used per-task in `onefuzz status top`.  [#1169](https://github.com/microsoft/onefuzz/pull/1169)
```

* Deployment: Application credentials created during deployment are no longer logged.  [#1172](https://github.com/microsoft/onefuzz/pull/1172)
* Deployment: Clarify logging when retrying AAD interactions.  [#1173](https://github.com/microsoft/onefuzz/pull/1173)
* Deployment: Replaced custom Azure Storage Queue creation with ARM templates.  [#1193](https://github.com/microsoft/onefuzz/pull/1193)

> * Service: The acceptable duration for SAS URLs is now back-dated to avoid time synchronization issues.  [#1195](https://github.com/microsoft/onefuzz/pull/1195)

Reviewer: ```suggestion
* Service: The validity period for SAS URLs is now back-dated to avoid time synchronization issues.  [#1195](https://github.com/microsoft/onefuzz/pull/1195)
```
